ceremonies/src/lib.rs

```diff
@@ -172,14 +172,14 @@ decl_module! {
         }
 
         #[weight = 10_000]
-        pub fn register_attestations(origin, attestations: Vec<Attestation<T::Signature, T::AccountId, T::Moment>>) -> DispatchResult {
+        pub fn attest_claims(origin, claims: Vec<ClaimOfAttendance<T::Signature, T::AccountId, T::Moment>>) -> DispatchResult {
             debug::RuntimeLogger::init();
             let sender = ensure_signed(origin)?;
             ensure!(<encointer_scheduler::Module<T>>::current_phase() == CeremonyPhaseType::ATTESTING,
-                "registering attestations can only be done during ATTESTING phase");
+                "registering claims can only be done during ATTESTING phase");
             let cindex = <encointer_scheduler::Module<T>>::current_ceremony_index();
-            ensure!(!attestations.is_empty(), "empty attestations supplied");
-            let cid = attestations[0].claim.community_identifier;
+            ensure!(!claims.is_empty(), "empty claims supplied");
+            let cid = claims[0].community_identifier;
```

Comment:
I have just come across this one.  This is potentially a new threat. If one claimant provides a wrong cid, all other claims are invalidated. Or shall we handle this in app? The claims are transparently stored as `ClaimOfAttendances`. We could easily discard invalid claims.